codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/shapevisitor/AwsSdkToDafnyShapeVisitor.java

```diff
@@ -120,20 +120,16 @@ public String blobShape(final BlobShape shape) {
       someWrapIfRequired = "Wrappers.Companion_Option_.Create_Some_(%s)";
       returnType = "Wrappers.Option";
     }
+    String bytesToDafnySequence = "dafny.SeqOfBytes(%s)".formatted(dataSource);
```

Comment:
Why not inline this directly in the outer formatted text?

Reply:
I thought that doing `someWrapIfRequired.formatted("dafny.SeqOfBytes(%s)".formatted(dataSource)` makes things confusing

Reply:
Great looks like this conversation have be resolved.